Python 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> import sqlite3
>>> conn = sqlite3.connect('enterprise.db')
>>> curs = conn.cursor()
>>> curs.execute('''CREATE TABLE hlth
    (region VARCHAR(20) PRIMARY KEY,
      incr FLOAT)''')

Traceback (most recent call last):
  File "<pyshell#5>", line 3, in <module>
    incr FLOAT)''')
OperationalError: table hlth already exists
>>> curs.execute('''CREATE TABLE hlthincr
    (region VARCHAR(20) PRIMARY KEY,
     incr FLOAT)''')
<sqlite3.Cursor object at 0x0000000001E4FCE0>
>>> ins = 'INSERT INTO hlthincr(region, incr) VALUES(?,?)
SyntaxError: EOL while scanning string literal
>>> ins = 'INSERT INTO hlthincr(region, incr) VALUES(?,?)'
>>> curs.execute(ins,('New England', 18.25))
<sqlite3.Cursor object at 0x0000000001E4FCE0>
>>> curs.execute(ins,('Mideast', 21.97))
<sqlite3.Cursor object at 0x0000000001E4FCE0>
>>> curs.execute(ins('Great Lakes', 2.79))

Traceback (most recent call last):
  File "<pyshell#14>", line 1, in <module>
    curs.execute(ins('Great Lakes', 2.79))
TypeError: 'str' object is not callable
>>> curs.execute(ins,('Great Lakes', 24.79))
<sqlite3.Cursor object at 0x0000000001E4FCE0>
>>> curs.execute(ins,('Plains', 24.36))
<sqlite3.Cursor object at 0x0000000001E4FCE0>
>>> curs.execute(ins,('Rocky Mountains', 35.87))
<sqlite3.Cursor object at 0x0000000001E4FCE0>
